packages/core/tests/mocks/MockWallet.ts

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import type { Wallet } from '../../src'
+import type { EncryptedMessage, WalletConfig, WalletExportImportConfig, WalletConfigRekey } from '../../src/types'
+import type { Buffer } from '../../src/utils/buffer'
+import type { DidInfo, UnpackedMessageContext, DidConfig } from '../../src/wallet'
+
+export class MockWallet implements Wallet {
+  public publicDid = undefined
+  public isInitialized = true
+  public isProvisioned = true
+
+  public create(walletConfig: WalletConfig): Promise<void> {
+    throw new Error('Method not implemented.')
```

Comment:
Are the not defined on purpose and jest will be used with mocking?

Reply:
These are not defined on purpose indeed. A lot of tests we just mock the calls to the wallet and it shouldn't call the actual methods.